### PR TITLE
Improve documentation + misc fixes

### PR DESCRIPTION
**Pull request by tas50**
Improve the formatting of the readme and better document more advanced shellout
Update travis config to a more modern config
Add badges to the readme
Opscode -> Chef
Copyright updates

### CODE REVIEW

**Comment by tas50** [comment]
@chef/client-core This is ready for review

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,13 @@
+language: ruby
+cache: bundler
+
+sudo: false
+
 rvm:
   - 1.9.3
   - 2.0.0
   - 2.1.2
+  - 2.2
```
Comment (danielsdeleo) [inline .travis.yml]: Does 2.3 also work?

File: README.md [hunk 2]
```diff
@@ -1,54 +1,87 @@
...
 
 Raise an exception if it didn't exit with 0
 
-      find.error!
+```ruby
+  find.error!
+```
+
+### Advanced Shellout
+In addition to the command to run there are other options that can be set to change the shellout behavior. The complete list of options can be found here: https://github.com/chef/mixlib-shellout/blob/master/lib/mixlib/shellout.rb
```
Comment (danielsdeleo) [inline README.md]: Might be worth calling out the behavior of `ENV` specifically? I agree with your reasoning for delegating to the code that has the exhaustive list, but env seems important for a lot of usage and it's behavior is a little tricky.

**Comment by danielsdeleo** [comment]
:+1: everything I commented on is minor enough you don't need to fix if you disagree.

File: .travis.yml (hunk 1 above)
Comment (tas50) [inline .travis.yml]: probably...

Comment (lamont-granquist) [inline .travis.yml]: should probably just be:

- 2.0
- 2.1
- 2.2
- 2.3

and let rvm pick the latest, along with dropping 1.9.3 support

**Comment by lamont-granquist** [comment]
:+1:

**Pull request merged**